core/src/test/scala/org/apache/pekko/persistence/jdbc/journal/dao/LimitWindowingStreamTest.scala

```diff
@@ -20,13 +20,14 @@
 package org.apache.pekko.persistence.jdbc.journal.dao
 
 import com.typesafe.config.{ ConfigValue, ConfigValueFactory }
-import org.apache.pekko.persistence.jdbc.journal.dao.LimitWindowingStreamTest.fetchSize
-import org.apache.pekko.persistence.jdbc.query.{ H2Cleaner, QueryTestSpec }
-import org.apache.pekko.persistence.{ AtomicWrite, PersistentRepr }
-import org.apache.pekko.stream.scaladsl.{ Keep, Sink, Source }
-import org.apache.pekko.stream.{ Materializer, SystemMaterializer }
+import org.apache.pekko
```

Comment:
vould you keep this with other pekko imports?

Reply:
fixed